src/core/src/bootstrap/Constants.py

```diff
@@ -271,8 +270,3 @@ class EnvLayer(EnumBackport):
         PRIVILEGED_OP_REBOOT = PRIVILEGED_OP_MARKER + "Reboot_Exception"
         PRIVILEGED_OP_EXIT = PRIVILEGED_OP_MARKER + "Exit_"
 
-    # These are numbers instead of strings like extension because they are customer facing in the error message diagnostics
-    class AgentEnvVarStatusCode(EnumBackport):
```

Comment:
nit: Ensure references to any of these codes in not left over in code

Reply:
Done, none left in core